skins/cat17/src/scripts/donationForm.js

```diff
@@ -442,12 +379,11 @@ $( function () {
 
   // connect DOM elements to actions
   // fixme don't use interval, use form events instead? Discuss performance & other implications
-  //$( '#continueFormSubmit1' ).click( WMDE.StoreUpdates.makeEventHandlerWaitForAsyncFinish( handlePaymentDataSubmit, store ) );
   $('input').on('click, change', WMDE.StoreUpdates.makeEventHandlerWaitForAsyncFinish( handleGroupValidations, store ) );
   setInterval(handleGroupValidations, 1000);
```

Comment:
Totally unrelated to this PR, but caught my eye because of the proximity to line 447: We should get rid of this interval stuff & handle async validation better, see https://phabricator.wikimedia.org/T181140